source/tests/pt/test_tabulate_fusion_se_a.py

```diff
@@ -0,0 +1,1461 @@
+                -0.05179508,
+                -0.03552708,
+                -0.02812173,
+                -0.04451295,
+                -0.04586229,
+                -0.03794369,
+                -0.02917727,
+                -0.04478649,
+            ],
+            device=env.DEVICE,
+        ).reshape(4, 4)
+        self.expected_dy_dem = torch.tensor(
+            [
+                -3.32965609,
+                -3.32965609,
+                -3.32965609,
+                -3.32965609,
+                -3.33781886,
+                -3.33781886,
+                -3.33781886,
+                -3.33781886,
+                -3.33501296,
+                -3.33501296,
+                -3.33501296,
+                -3.33501296,
+                -3.34559974,
+                -3.34559974,
+                -3.34559974,
+                -3.34559974,
+                -3.34561821,
+                -3.34561821,
+                -3.34561821,
+                -3.34561821,
+                -3.33106684,
+                -3.33106684,
+                -3.33106684,
+                -3.33106684,
+                -3.33218328,
+                -3.33218328,
+                -3.33218328,
+                -3.33218328,
+                -3.34535585,
+                -3.34535585,
+                -3.34535585,
+                -3.34535585,
+                -3.34250754,
+                -3.34250754,
+                -3.34250754,
+                -3.34250754,
+                -3.33356685,
+                -3.33356685,
+                -3.33356685,
+                -3.33356685,
+                -3.33652989,
+                -3.33652989,
+                -3.33652989,
+                -3.33652989,
+                -3.3309235,
+                -3.3309235,
+                -3.3309235,
+                -3.3309235,
+                -3.33688909,
+                -3.33688909,
+                -3.33688909,
+                -3.33688909,
+                -3.34206038,
+                -3.34206038,
+                -3.34206038,
+                -3.34206038,
+                -3.34387412,
+                -3.34387412,
+                -3.34387412,
+                -3.34387412,
+                -3.33051143,
+                -3.33051143,
+                -3.33051143,
+                -3.33051143,
+            ],
+            device=env.DEVICE,
+        ).reshape(4, 4, 4)
+
+    def test_forward(self):
+        # Call the forward function
+        forward_result = torch.ops.deepmd.tabulate_fusion_se_a(
+            self.table_tensor,
+            self.table_info_tensor,
+            self.em_x_tensor,
+            self.em_tensor,
+            self.last_layer_size,
+        )
+
+        descriptor_tensor = forward_result[0]
+
+        # Check the shape
+        self.assertEqual(descriptor_tensor.shape, self.expected_descriptor_tensor.shape)
+
+        # Check the values
+        torch.testing.assert_close(
+            descriptor_tensor, self.expected_descriptor_tensor, atol=1e-5, rtol=1e-5
```

Comment:
The current test is for `float32`. We may test **both** `float32` and `float64` and ensure `float64` has an accuracy of 1e-10.

Reply:
The dtype should be explicitly given, such as `dtype = torch.float64`.

Reply:
I have no idea. I expect @nahso could explain it.

Reply:
We think that the test samples in `tabulate_fusion_se_a.cc` cannot support the 1e-10 level test.  For example, the descriptor has a precision of only 7 decimal places, so the test only passes the test at the 1e-7 level and fails at 1e-8.
So I ran the results directly as the expected results and all tests passed now.